Header header
int32 CommunicationState
bool PadConnected
int32 ControlMode